content/backend/graphql-js/7-subscriptions.md

```diff
@@ -123,17 +185,26 @@ module.exports = {
 
 </Instruction>
 
-The code seems pretty straightforward. As mentioned before, the subscription resolver is provided as the value for a `subscribe` field inside a plain JavaScript object.
+The code seems pretty straightforward. As mentioned before,
+the subscription resolver is provided as the value for a
+`subscribe` field inside a plain JavaScript object.
 
-Now you can see how we access `pubsub` on the `context` and call `asyncIterator()` passing the string `"NEW_LINK"` into it. This function is used to resolve subscriptions and push the event data.
+Now you can see how we access `pubsub` on the `context` and
+call `asyncIterator()` passing the string `"NEW_LINK"` into
+it. This function is used to resolve subscriptions and push
+the event data.
 
 ### Adding subscriptions to your resolvers
 
-The last thing we need to do for our subscription implementation itself is to actually call this function inside of a resolver!
+The last thing we need to do for our subscription
+implementation itself is to actually call this function
+inside of a resolver!
 
 <Instruction>
 
-Pop over to `Mutation.js` and locate your `post` resolver function, adding the following call to `pubsub.publish()` right before we return our `newLink`:
+Pop over to `Mutation.js` and locate your `post` resolver
+function, adding the following call to `pubsub.publish()`
+right before we return our `newLink`:
 
 ```js(path=".../hackernews-node/src/resolvers/Mutation.js")
```

Comment:
I think it would be nice to show the user what changed in the Mutation 🙂
```suggestion
```js{4,11,13}(path=".../hackernews-node/src/resolvers/Mutation.js")
```